packages/parser/common_tx_parser.go

```
@@ -141,9 +141,9 @@ func (p *Parser) DeleteQueueTx(hash []byte) error {
 // AllTxParser parses new transactions
 func (p *Parser) AllTxParser() error {
 	logger := p.GetLogger()
-	all, err := model.GetAllUnverifiedAndUnusedTransactions()
+	all, _ := model.GetAllUnverifiedAndUnusedTransactions()
```

again, ignoring errors are bad